src/lib/index.jsx

```diff
@@ -29,65 +29,64 @@ export default function AddToCalendar(WrappedButton, WrappedDropdown) {
       className: null,
       dropdownProps: {},
       items: Object.values(SHARE_SITES),
-      linkProps: {},
+      linkProps: {}
     };
 
     state = {
-      dropdownOpen: false,
+      dropdownOpen: false
     };
 
-    handleCalendarButtonClick = e => {
+    handleCalendarButtonClick = (e) => {
       e.preventDefault();
       const url = e.currentTarget.getAttribute('href');
       if (url.startsWith('BEGIN')) {
         const filename = 'download.ics';
         const blob = new Blob([url], { type: 'text/calendar;charset=utf-8' });
 
-        const link = document.createElement('a');
-        link.href = window.URL.createObjectURL(blob);
-        link.setAttribute('download', filename);
-        document.body.appendChild(link);
-        link.click();
-        document.body.removeChild(link);
+        if (isInternetExplorer()) {
+          // IE 11 approach
+          window.navigator.msSaveOrOpenBlob(blob, filename);
+        } else {
+          const link = document.createElement('a');
+          link.href = window.URL.createObjectURL(blob);
+          link.setAttribute('download', filename);
+          document.body.appendChild(link);
+          link.click();
+          document.body.removeChild(link);
+        }
       } else {
-        window.open(url, '_blank');
+        // Mobile download
+        // _self = iOS compatible. Android supports either _self or _blank it seems.
```

Comment:
Good info, but we don't need this comment. You can convey this info in the PR, but it's superfluous in the code.

Reply:
Also, I think `_self` vs `_blank` should be a prop passed in by the user. I tested this on iOS 11.4.1 and have no issues opening the calendar in a new tab -- it doesn't close the dialog.